### ISSUE TITLE
AFX 1.5.4 - extra newlines in text copied from AsciidocFX

### ISSUE THREAD

**Issue by wfouche** [asciidocfx/AsciidocFX#265]
Hi @rahmanusta,

When copying text from AFX each line of text has an extra newline character. For example, copying text:

```
[plantuml, network-diagram, png]
----
@startdot
digraph G {
    node [shape=box]
	subgraph cluster_0 {
		node [style=filled];
		WIN_11 -> WIN_12 -> WIN_13;
		label = "WIN - Site 1";
		color = purple
	}

	subgraph cluster_1 {
		node [style=filled];
		WIN_21 -> WIN_22 -> WIN_23;
		label = "WIN - Site 2";
		color = purple
	}
	start -> WIN_11;
	start -> WIN_21;
	WIN_13 -> end;
	WIN_23 -> end;
    
    users [shape=Mdiamond]
    users -> start
    start [label="WIN_00", shape=invhouse]
	end [label="Peer-to-Peer\nReplication", shape=Msquare];
}
@enddot
----
```

from AsciidocFX to Notepad++ results in

```
[plantuml, network-diagram, png]

----

@startdot

digraph G {

    node [shape=box]

	subgraph cluster_0 {

		node [style=filled];

		WIN_11 -> WIN_12 -> WIN_13;

		label = "WIN - Site 1";

		color = purple

	}



	subgraph cluster_1 {

		node [style=filled];

		WIN_21 -> WIN_22 -> WIN_23;

		label = "WIN - Site 2";

		color = purple

	}

	start -> WIN_11;

	start -> WIN_21;

	WIN_13 -> end;

	WIN_23 -> end;

    

    users [shape=Mdiamond]

    users -> start

    start [label="WIN_00", shape=invhouse]

	end [label="Peer-to-Peer\nReplication", shape=Msquare];

}

@enddot

----
```

being displayed in Notepad++ on Windows.

Regards,

Werner

**Comment by rahmanusta** [comment]
Could you please test v1.5.5?

**Comment by wfouche** [comment]
Fixed confirmed for AFX 1.5.5.
